client/src/redux/PostsRedux.js

```diff
@@ -97,3 +104,16 @@ export const loadSinglePostRequest = id => {
     }
   };
 };
+
+export const addPostRequest = post => {
+  return async dispatch => {
+    dispatch(startRequest());
+    try {
+      let res = await axios.post(`${API_URL}/posts`, post);
+      await new Promise((resolve, reject) => setTimeout(resolve, 2000));
```

Comment:
Co to za sztuczny opóźniacz? Czemu to ma służyć?

Reply:
w zaadniu - sprawdzenie czy działa asynchroniczność

Reply:
Rozumiem sytuację kiedy nie ma komunikacji z serwerem api, żeby zrobić sobie taką symulację,  ale masz juz komunikację z api,  nie jest to juz potrzebne,  usuń to proszę

Reply:
jasne, z checią!

Reply:
Już wiem, by pokazać, ze spinner działa - teraz za szybko leci by się zdążył wyświetlić

Reply:
Na potrzeby testu spinnera ok chociaż wolałbym bym dać takie opóźnienie dla testu po stronie serwera,  jak przetestowane że działa to usuń ten kod,  pamietaj działasz lokalnie tu nie ma opóźnienie praktycznie,  a potem robisz deploy takiej aplikacji i nagłe spinner działa,  bo w necie są opóźnienie,  a Twój użytkownik musiał by czekać dodatkowo 2 sekundy,..

Reply:
Przetestowane. Usunięte.